### DISCUSSION TITLE
DITA-OT 4.0

### DISCUSSION THREAD

**Discussion by infotexture** [dita-ot/dita-ot#4051] [Announcements]
### DITA-OT 4.0 has been released

DITA-OT 4.0 was announced yesterday at [DITA-OT Day](https://www.oxygenxml.com/events/2022/dita-ot_day.html) in Rotterdam:

> DITA-OT 4.0 requires Java 17 and includes a new plug-in for easier PDF customization, project file improvements, updates to LwDITA processing, and support for the split chunking feature in the latest drafts of the upcoming DITA 2.0 standard.

✨  For details, see the _Release Notes_ at [dita-ot.org/4.0/release-notes](https://www.dita-ot.org/4.0/release-notes/).

⬇️ The [distribution package](https://github.com/dita-ot/dita-ot/releases/download/4.0/dita-ot-4.0.zip) is available from the [Downloads](https://www.dita-ot.org/download) page on the project website.

**Comment by raducoravu** [comment]
Is there any intentional incompatible change between DITA OT 3.7.4 and 4.0? Besides the Java VM version jump....

**Comment by infotexture** [comment]
@raducoravu We've noted a few other things that you might want to be aware of in the [4.0 migration topic](https://www.dita-ot.org/dev/topics/migrating-to-4.0.html).

ℹ️ Depending on how you've set things up in your source files and customizations, you may want to be aware of the changes to attribute set reflection and default encoding for code references as mentioned there.
